config/i18n.js

```diff
@@ -1,5 +1,13 @@
-export default {
+const clientI18n = {
   langPickerRender: true,
   langList: ['en-US', 'ru-RU'],
   fallbackLng: 'en-US'
 };
+
+const serverI18n = {
```

Comment:
We shouldn't have per-platform config here. Did you see that any of the config files in the kit are per-platform? No, there is no per-platform files, each config is for all platforms. So, don't do it here as well. Reduce the number of config settings to minimum. Why you have `locales` and `langList` here it is the same thing, remove `locales`, use `langList`, etc.